tests/poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
although our CI is installing Poetry 1.8.5, not sure if it could be a problem

https://github.com/qdrant/qdrant/blob/master/.github/workflows/integration-tests.yml#L11